File: Contains Duplicate (easy)/Contains Duplicate (easy).py
class Solution:
    def containsDuplicate(self, nums):
      for i in range (len(nums)):
        for j in range(i + 1, len(nums)):
          if nums[i]==nums[j]:
            return True
          else:
            continue 

      return False
    


if __name__ == '__main__':
  sol = Solution()
  nums1 = [1, 2, 3, 4]
  print(sol.containsDuplicate(nums1)) 

File: Contains Duplicate (easy)/Contains Duplicate (easy).cpp
#include <iostream>
#include <vector>
#include <unordered_set> // Include the necessary header for unordered_set

using namespace std;

class Solution
{
public:
    bool containsDuplicate(vector<int> &nums)
    {
        unordered_set<int> seen;
        for (int num : nums)
        {
            if (seen.count(num) > 0)
            {
                return true;
            }
            seen.insert(num);
        }
        return false;
    }
};

int main()
{
    Solution sol1;
    vector<int> nums1 = {1, 2, 2, 4};

    cout << sol1.containsDuplicate(nums1) << endl;

    return 0;
}